If you try to build without opaque LSA support you'll hit this error. I now
turn this on in the makefile so this isn't really needed but it's here just
because I fixed it before changing the configuration.

--- quagga-0.99.2/ospfd/ospfd.c.orig	2005-11-13 13:28:39.000000000 +1100
+++ quagga-0.99.2/ospfd/ospfd.c	2005-11-13 13:28:57.000000000 +1100
@@ -466,7 +466,9 @@
   OSPF_TIMER_OFF (ospf->t_lsa_refresher);
   OSPF_TIMER_OFF (ospf->t_read);
   OSPF_TIMER_OFF (ospf->t_write);
+#ifdef HAVE_OPAQUE_LSA
   OSPF_TIMER_OFF (ospf->t_opaque_lsa_self);
+#endif
 
   close (ospf->fd);
   stream_free(ospf->ibuf);
